xgitguard/github-enterprise/enterprise_cred_detections.py

```diff
@@ -224,6 +224,8 @@ def format_detection(skeyword, org_url, url, code_content, secrets, skeyword_cou
                 else:
                     # Mask the current secret
                     masked_secret = mask_data(code_line, secret)
+                linenumber=findLineNumber(code_content.split("\n"),code_line)
```

Comment:
let's rename variable since it is is not the line number, rather a list of code lines, Also we can move this line to the utility function.